modules/mobile/docs/openapi.yaml

```diff
@@ -3439,6 +3439,7 @@ paths:
                   meta:
                     properties:
                       availableServices:
+                        description: Services that the app has through MobileAPI.
```

Comment:
Can we change this to something like `All services available through the mobile API. See authorizedServices for a list of services available to the user.`

Reply:
updated!